src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java

```diff
@@ -16973,6 +16973,27 @@ public void run3() {
                 ItemList.Circuit_Parts_CapacitorSMD.get(48L),
                 320,
                 120);
+
+        GT_Values.RA.addAssemblerRecipe(
+                new ItemStack[] {
+                    GT_OreDictUnificator.get(OrePrefixes.ring, Materials.Neodymium, 1),
+                    GT_OreDictUnificator.get(OrePrefixes.wireFine, Materials.Platinum, 2),
```

Comment:
platinum is already used for diode, also it is also heavily used in other circuit compents. personally i suggest to use fine conductive iron wire

Reply:
Replacing platinum could work, the only concern is that the recipe should be in line with the cost of the other SMDs. Maybe if it gets too easy using e.g. fine conductive iron the yield should be 24 and 48 instead?

Reply:
smd is unlocked in mv so i don't think we should make it too difficult. but yea conductive iron outputing 32 inductor is too much. conductive iron with 24 inductor should be more suitable